### PR TITLE
52367-add-admin-routes-and-pages

### PR DESCRIPTION
**Pull request by l0ngvh**
## Related Tickets
- [#52367](https://edu-redmine.sun-asterisk.vn/issues/52367)

## WHAT (optional)
- I added an admin dashboard page

## Evidence (Screenshot or Video)
image

### CODE REVIEW

File: app/components/course_card_component/course_card_component.scss [hunk 1]
```diff
@@ -1,5 +1,3 @@
-@import "colors";
-
 .course-card {
   .card-body {
     background-color: #f1eee9;
```
Comment (htpt2910) [inline app/components/course_card_component/course_card_component.scss]: đưa mã màu vào colors

**Review by htpt2910 — COMMENTED** [review]
cmt

**Comment by l0ngvh** [comment]
ready

File: app/components/admin/dash_board_component.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class Admin::DashBoardComponent < ViewComponent::Base
+  def initialize
+    super
+    @courses = Course.first 4
```
Comment (ngocvt-0484) [inline app/components/admin/dash_board_component.rb]: cho 4 vào setting

**Comment by l0ngvh** [comment]
ready

**Comment by l0ngvh** [comment]
ready

**Pull request merged**